charts/generic-service/values.yaml

```
@@ -29,6 +29,15 @@ config: {}
 configMountPath: /config/config.yaml
 additionalConfigs: []
 
+# dnsConfig:
+#   nameservers:
+#     - 192.0.2.1
+#   searches:
+#     - my.dns.search.suffix
+#   options:
+#     - name: ndots
+#       value: "2"
```

We usually just put "empty" default values here, which helps with some GUIs like: https://artifacthub.io/packages/helm/nano-byte/generic-service?modal=values-schema

```suggestion
dnsConfig: {}
```